client/app/scripts/superdesk-authoring/authoring.js

```diff
@@ -687,71 +688,60 @@
         };
     }
 
-    CropImageService.$inject = ['urls', '$http'];
-    function CropImageService(urls, $http) {
-        this.saveCrop = function saveCrop(coordinates, item) {
-            return urls.resource('archive').then(function(url) {
-                url = url + '/' + item._id + '/crop/' + item.cropsize.name;
-                return $http.post(url, {
-                    'CropLeft': coordinates.CropLeft,
-                    'CropRight': coordinates.CropRight,
-                    'CropTop': coordinates.CropTop,
-                    'CropBottom': coordinates.CropBottom
-                }).then(function(response) {
-                    return response.data;
-                });
-            });
-        };
-    }
-
-    ChangeImageController.$inject = ['$scope', 'cropImage', 'gettext', 'notify', '$route'];
-    function ChangeImageController($scope, cropImage, gettext, notify, $route) {
+    ChangeImageController.$inject = ['$scope', 'gettext', 'notify', 'modal', '$q'];
+    function ChangeImageController($scope, gettext, notify, modal, $q) {
         $scope.data = $scope.locals.data;
         $scope.preview = {};
-
-        $scope.close = function() {
-            if ($scope.data.state !== 'published') {
-                $scope.reject();
-                $route.reload();
-            } else {
-                return $scope.resolve($scope.data);
+        $scope.origPreview = {};
+        $scope.data.cropData = {};
+        $scope.data.isDirty = false;
+
+        $scope.$watch('preview', function(newValue, oldValue) {
+            if (newValue === oldValue) {
+                $scope.origPreview = $scope.preview;
+                return;
             }
-        };
+            if (Object.keys(newValue).length === Object.keys(oldValue).length) {
```

Comment:
I suggest adding an explanation why this check is in place, it is probably not obvious for people not knowing all the details.